jiant/tasks/tasks.py

```diff
@@ -2753,12 +2754,11 @@ def load_data(self):
                     self.tokenizer_name, filename, has_labels=False
                 )
             else:
-                iters_by_split[split] = load_span_data(self.tokenizer_name, filename)
+                iters_by_split[split] = load_span_data(
+                    self.tokenizer_name, filename, label_fn=lambda x: 0 if x is False else 1
+                )
         self._iters_by_split = iters_by_split
 
-    def get_all_labels(self):
-        return ["False", "True"]
```

Comment:
Make sure that the prediction files created in `evaluate.py` still have whatever the output labels should be

Reply:
yep checked!